src/components/SDKsPage/MainSection/index.tsx

```diff
@@ -7,27 +7,34 @@ import Link from 'src/components/Link';
 export enum Tab {
   CHANNELS = 'channels',
   SPACES = 'spaces',
+  CHAT = 'chat',
 }
 
 const MainSection = ({ tab }: { tab: Tab }) => {
-  const activeTab = tab !== Tab.SPACES ? Tab.CHANNELS : Tab.SPACES;
+  const activeTab = tab === Tab.CHAT ? Tab.CHAT : tab === Tab.SPACES ? Tab.SPACES : Tab.CHANNELS;
 
   return (
     <div>
       <div>
         <div className={`${container}`}>
           <Link
             to="/sdks"
-            className={`ui-text-h3 mr-16 px-8 py-16 inline-block ${activeTab === Tab.CHANNELS ? active_tab : null}`}
+            className={`text-h3 mr-16 px-8 py-16 inline-block ${activeTab === Tab.CHANNELS ? active_tab : null}`}
           >
             Pub/Sub
           </Link>
           <Link
             to="/sdks?tab=spaces"
-            className={`text-h3 font-normal px-8 py-16 inline-block ${activeTab === Tab.SPACES ? active_tab : null}`}
+            className={`text-h3 mr-16 px-8 py-16 inline-block ${activeTab === Tab.SPACES ? active_tab : null}`}
```

Comment:
We can retire the usage of this CSS module here and stick with Tailwind which integrates in with our design system a little better. In Tailwindish:

- `font-weight: 800` = `font-extrabold`
- `color: #ff5416` = `text-orange-600`
- I have no clue what role `relative` would play here, it can probably be removed, but for parity that would just be `relative`

Ideally we can then also delete the `.active_tab` class.

Further, not needed here but [take a look at the `cn` helper we use in `docs`](https://github.com/ably/docs/blob/main/src/components/Layout/Layout.tsx#L96-L98) to make constructing class strings a bit more streamlined instead of interpolating.

Reply:
Thanks for the tips and the info - looks a lot cleaner 🙂

Reply:
I see that `active_tab` is still being imported and referenced but it's been deleted from the imported CSS module file, I'd imagine this would cause problems - I'd say you'd need to go a bit further and add in the other classes listed above (unless they are no longer needed) and then remove the import to the now-removed `active_tab` class

Reply:
i.e. something like
```ts
className={`ui-text-h3 mr-16 px-8 py-16 inline-block ${activeTab === Tab.SPACES ? 'relative font-extrabold text-orange-600' : null}`}
```

Reply:
Oops, you're quite right. Shouldn't have been multi-tasking... Tidied this up now.

Reply:
Apologies, I didn't see the `::after` part of the now-departed CSS module (which forms the underline under the selected tab) - we would lose the underline without porting that as well. To save more PR tennis I've popped this in a followup commit here. One could argue this isn't a save now at all, but hey, at least it brings the styling approach more towards "standard".